### PR TITLE
PEM Docs: Adding missing OLH topic content to the Doc topic

### PR DESCRIPTION
**Pull request by arup-roy2023**
## What Changed?

Content from the multiple Online Help topics that are missing from the Docs topics is being added to them as part of this PR

### CODE REVIEW

**Comment by drothery-edb** [comment]
Are we sure we want to move this into the main content? Referring users to the actual OLH seems sufficient to me.

**Comment by arup-roy2023** [comment]
Hi Dee Dee,
Currently, the Docs topic '[Managing configuration settings](https://www.enterprisedb.com/docs/pem/latest/managing_configuration_settings/)' includes the following line: "A copy of the default config.py file is included in the PEM online help for reference." The online help it is referencing is the '[Managing Configuration Settings](https://www.enterprisedb.com/docs/pem/latest/pem_online_help/01_toc_pem_getting_started/03_pem_managing_configuration_settings/)' which includes the default config.py file content. As the Managing Configuration Settings topic in Docs' Online Help is being removed as part of the Remove redundant content from the OLH task, I added the config.py file content and the redirect link to the Managing configuration settings topic in Docs.
Please provide your inputs about the config.py file content availability in the Managing configuration settings topic in Docs.

File: product_docs/docs/pem/9/tuning_performance/tuning_wizard.mdx [hunk 1]
```diff
@@ -14,68 +14,67 @@ Before using the Tuning Wizard, you must specify the name of the service in the
...
+1. To open the Tuning Wizard, in the PEM client select **Management > Tuning Wizard**.
 
-1.  Select **Next**.
+2. Select **Next**.
 
-1.  When you expand the **Servers** node of the tree, a list of servers appears. All of these servers are currently monitored by PEM and available for tuning. Select a server to tune it. 
+3. When you expand the **Servers** node of the tree, a list of servers appears. All of these servers are currently monitored by PEM and available for tuning. Select a server to tune it. 
 
-   !!! Note
+   !!! Note Note
```
Comment (nidhibhammar) [inline product_docs/docs/pem/9/tuning_performance/tuning_wizard.mdx:23]:
```suggestion
   !!! Note
```

File: product_docs/docs/pem/9/monitoring_performance/log_manager.mdx [hunk 2]
```diff
@@ -220,3 +231,95 @@ The report displays the tables, graphs, and charts selected in the Log Analysis
...
+-   The Most Time Executed Query Statistics table displays the queries that took the most execution time on the server. The table will include the number of entries specified in the **Rows Limit** field of the Log Analysis Expert.
+    -   The Query column displays the text of the performed query.
+    -   The Parameters column displays the parameters (if the query is a parameterized query).
+    -   The No. of Times Executed column displays the number of times that the query executed.
+    -   The Total Duration column displays the length of time that it took the server to execute the query.
+-   The Connections Overview Timeline section of the Log Analysis Expert report displays information about successful and unsuccessful connection attempts during the specified time period:
+    -   The Timestamp graph displays the number of server connections attempted and connections authenticated at any given point during the specified time period. Hover over a specific point to view detailed information about that point on the graph.
+    -   The Summary pie chart displays the relative percentage of connections attempted and connections authenticated during the specified time period.
```
Comment (ebgitelman) [inline product_docs/docs/pem/9/monitoring_performance/log_manager.mdx:325]: @drothery-edb We should consider a different format for this. Or perhaps just pull out the first level bullet to a regular paragraph.

**Review by nidhibhammar — APPROVED** [review]

**Pull request merged**